components/Card.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react/no-danger */
+
+import React from 'react';
+import Link from 'next/link';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+export default function Card(props) {
+  const { thumbnail, title, excerpt, layout, link } = props;
+
+  if (link) {
+    return (
+      <CardWrap className={`card card--${layout}`}>
+        {link && (
+          <Link href={link}>
+            <a className="flexwrap">
+              {thumbnail && (
+                <div
+                  className="card__thumbnail"
+                  style={{ backgroundImage: `url(static/images/thumbnails/${thumbnail})` }}
+                ></div>
+              )}
+              {title && <h3 className="card__title">{title}</h3>}
+              {excerpt && <p className="card__excerpt">{excerpt}</p>}
+            </a>
+          </Link>
+        )}
```

Comment:
We're already checking `if (link)` above, so I think we can clean this up a bit.

```suggestion
        <Link href={link}>
          <a className="flexwrap">
            {thumbnail && (
              <div
                className="card__thumbnail"
                style={{ backgroundImage: `url(static/images/thumbnails/${thumbnail})` }}
              ></div>
            )}
            {title && <h3 className="card__title">{title}</h3>}
            {excerpt && <p className="card__excerpt">{excerpt}</p>}
          </a>
        </Link>
```